client/src/components/SubsiteChangeAlert/SubsiteChangeAlert.jsx

```diff
@@ -0,0 +1,59 @@
+/* global window */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import i18n from 'i18n';
+import createEvent from 'legacy/createEvent';
+
+class SubsiteChangeAlert extends Component {
+  constructor(props) {
+    super(props);
+    this.handleRevert = this.handleRevert.bind(this);
+  }
+
+  handleRevert() {
```

Comment:
I'm finding the use of the word 'revert' throughout this confusing.  Shouldn't this be 'change' instead?

Reply:
No, revert describes what the intent is, not what the event is.